.gitlab/e2e/e2e.yml

```diff
@@ -118,22 +118,17 @@ k8s-e2e-otlp-main:
     E2E_COMMIT_SHA: $CI_COMMIT_SHORT_SHA
     E2E_OUTPUT_DIR: $CI_PROJECT_DIR/e2e-output
   script:
-    - inv -e new-e2e-tests.run --targets $TARGETS -c ddagent:imagePullRegistry=669783387624.dkr.ecr.us-east-1.amazonaws.com -c ddagent:imagePullUsername=AWS -c ddagent:imagePullPassword=$(aws ecr get-login-password) --junit-tar "junit-${CI_JOB_NAME}.tgz" ${EXTRA_PARAMS}
+    - inv -e new-e2e-tests.run --targets $TARGETS -c ddagent:imagePullRegistry=669783387624.dkr.ecr.us-east-1.amazonaws.com -c ddagent:imagePullUsername=AWS -c ddagent:imagePullPassword=$(aws ecr get-login-password) --junit-tar junit.tgz ${EXTRA_PARAMS}
   after_script:
     # Upload generated junit files
     - export DATADOG_API_KEY=$($CI_PROJECT_DIR/tools/ci/aws_ssm_get_wrapper.sh ci.datadog-agent.datadog_api_key_org2)
-    - for f in junit-new-e2e-*.tgz; do inv -e junit-upload --tgz-path "$f"; done
+    - inv -e junit-upload --tgz-path junit.tgz
   artifacts:
     expire_in: 2 weeks
     when: always
     paths:
-      # This file will be consumed by the `e2e_test_junit_upload` job in next stage to push the report to datadog.
-      # If you create a new job from this template, do not forget to update the `dependencies` of the `e2e_test_junit_upload` job.
-      - junit-*.tgz
```

Comment:
Could we want to keep archiving the artefacts for investigation?

Reply:
Done